app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
Please provide type annotation here

Reply:
Type annotation for what? For class attribute people? Why are there often no examples in the lessons and video theory...((( Thank you! I didn't know about type annotation for class atributes.